src/components/mc/cpu/mc_cpu.c

```diff
@@ -119,16 +119,9 @@ static ucc_status_t ucc_mc_cpu_memcpy(void *dst, const void *src, size_t len,
     return UCC_OK;
 }
 
-static ucc_status_t ucc_mc_cpu_mem_query(const void *ptr, size_t length,
-                                        ucc_mem_attr_t *mem_attr)
+static ucc_status_t ucc_mc_cpu_mem_query(const void *ptr, //NOLINT
+                                         ucc_mem_attr_t *mem_attr) //NOLINT
 {
-    if (ptr == NULL || length == 0) {
```

Comment:
What happens if we run CPU only env, alltoall + PRE_REG_MEM ? There will be a call to mem_query? Will we see errors?

Reply:
No errors because get_mem_attr skips cpu mc
```
 for (mt = UCC_MEMORY_TYPE_HOST + 1; mt < UCC_MEMORY_TYPE_LAST; mt++)
```